src/_layouts/header.njk

```diff
@@ -16,5 +16,11 @@
     <div id="menu-reveal" class="menu-button-container right">
       Menu&nbsp;<i class="fa fa-bars"></i>
     </div>
+    <div class="menu">
+        <a title="Home" href="/" class="menu-item"></a>
+        <a title="About Us" href="/#about-us"></a>
+        <a title="Residential" href="/#residential"></a>
+        <a title="Contact Us" href="/#contact-us"></a>
+    </div>
```

Comment:
We could actually make use of Nunjuck's built-in [for-loop](https://mozilla.github.io/nunjucks/templating.html#for) to make this a lot more simple, so we can make use of everything in the `menu.yml`